src/common/markdown.ts

```diff
@@ -26,7 +26,7 @@ export const markdown: MarkdownIt = MarkdownIt({
 export const getMentionLink = ({ id, username }: MarkdownMention): string => {
   const href = getUserProfileUrl(username || ghostUser.id);
 
-  return `<a href="${href}" data-mention-id="${id}" data-mention-username="${username}">@${username}</a>`;
+  return `<a href="${href}" data-mention-id="${id}" data-mention-username="${username}" translate="no">@${username}</a>`;
```

Comment:
this will be detected on kvasir?

Reply:
[Google Translate detects it](https://cloud.google.com/translate/troubleshooting#:~:text=How%20do%20I%20tell%20Cloud%20Translation%20%2D%20Advanced%20to%20not%20translate%20something%3F)